src/packet.cpp

```diff
@@ -7,13 +8,13 @@
 Packet::Packet(const std::array<uint8_t, 4096>& m_recvBuffer, size_t bytes, const std::string& secret)
 {
     if (bytes < 20)
-        throw std::runtime_error{"The number of received bytes in the request - " + std::to_string(bytes) + " is less than 20 bytes"};
+        throw std::runtime_error{"lib::Error::numberBytesLess20"};
```

Comment:
Instead of throwing `std::runtime_error`, please make a dedicated exception class, inherited from `std::runtime_error`. It must receive `Error` enum in the constructor, create an internal `error_code` member and use it to make a message for the `std::runtime_error` constructor.

Reply:
This line should look like this:
```c++
throw Exception(Error::numberBytesLessThan20);
```